nanoFrameworkExceptionTesting/Program.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Diagnostics;
+using System.Threading;
+
+namespace nanoFrameworkExceptionTesting
+{
+    public class Program
+    {
+        public static void Main()
+        {
+            Debug.WriteLine("Hello from nanoFramework!");
+
+            var stopWatch = new Stopwatch();
+            stopWatch.Start();
+
+            byte[] dummyArray = new byte[10];
+
+            try
+            {
+                // throw Exception
+                Debug.WriteLine($"{dummyArray[1_000]}");
```

Comment:
I don't think this code line will run in Release build. Is that desired?

Reply:
You are correct. Replaced with Console.